analysis/src/SharedTypes.ml

```diff
@@ -6,7 +6,7 @@ type modulePath =
   | File of Uri.t * string
   | NotVisible
   | IncludedModule of Path.t * modulePath
-  | ExportedModule of string * modulePath
+  | ExportedModule of string * modulePath * bool
```

Comment:
This is getting a bit confusing above, what about an inline record so the 3 components of the payload are named.

Reply:
Done